app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -205,7 +205,7 @@ def quota_find_provisions(options)
   def quota_find_provision_by_owner(options)
     email = get_option(:owner_email).to_s.strip
     sched_type = get_option(:schedule_type).to_s.strip
-    options[:scheduled_time] = sched_type == 'schedule' ? get_option(:schedule_time) : nil
+    options[:scheduled_time] = get_option(:schedule_time) if sched_type == 'schedule'
```

Comment:
This is actually different.  Previous the value was explicitly set to nil, whereas the new code keeps the previous value.